thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/anomaly/utils/AbstractResourceHttpUtils.java

```diff
@@ -1,22 +1,35 @@
 package com.linkedin.thirdeye.anomaly.utils;
 
+import com.sun.xml.internal.ws.api.policy.PolicyResolver;
```

Comment:
this private dependency should not be here

Reply:
It's removed. Thanks for pointing out.